examples/spark/steps/importer_step/importer_step.py

```diff
@@ -0,0 +1,30 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from pyspark.sql import DataFrame, SparkSession
+
+from zenml.repository import Repository
+from zenml.steps import BaseStepConfig, step
+
+step_operator = Repository().active_stack.step_operator
```

Comment:
maybe here you could make sure that this really is a Spark step operator instead